tests/test_binding.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from anys import ANY_STR
+from test_helpers import *
+
+
+# TODO(#294): Implement E2E tests for binding.
+@pytest.mark.asyncio
+async def test_binding(websocket, context_id):
+    result = await execute_command(
+        websocket, {
+            "method": "script.callFunction",
+            "params": {
+                "functionDeclaration": "(binding)=>binding('SOME_MESSAGE')",
+                "arguments": [{
+                    "type": "PROTO.binding",
+                    "value": "SOME_BINDING"
+                }],
+                "target": {
+                    "context": context_id
+                },
+                "awaitPromise": False,
+                "resultOwnership": "root"
+            }
+        })
+
+    assert {
```

Comment:
(binding)=> <<<< setTimeout(binding('SOME_MESSAGE')) >>> (racy / flaky --> delay by ~1ms for now)

Reply:
Done